.ci/setup.sh

```diff
@@ -1,22 +1,13 @@
 #!/bin/bash
 #
-# Copyright (c) 2017 Intel Corporation
+# Copyright (c) 2017-2018 Intel Corporation
 #
 # SPDX-License-Identifier: Apache-2.0
 
 set -e
 
-test_repo="github.com/clearcontainers/tests"
+tests_repo="github.com/kata-containers/tests"
+tests_repo_dir="${GOPATH}/src/${tests_repo}"
 
-# Clone Tests repository.
-go get "$test_repo"
-
-test_repo_dir="${GOPATH}/src/${test_repo}"
-
-# Check the commits in the branch
-checkcommits_dir="${test_repo_dir}/cmd/checkcommits"
-(cd "${checkcommits_dir}" && make)
-checkcommits \
-	--need-fixes \
-	--need-sign-offs \
-	--verbose
+go get -d "$tests_repo" || true
```

Comment:
One day I will really confirm if we can use

```
got get -d -u "$tests_repo"/...
```

and avoid that whole `true` thing...

Reply:
I'm pretty sure `go get` is dumb and does `exit 1` even with `-d` if no golang source files are found, so the `|| true` is currently required.